edx_proctoring/templates/proctored_exam/expired.html

```diff
@@ -0,0 +1,20 @@
+{% load i18n %}
+<div class="sequence proctored-exam completed" data-exam-id="{{exam_id}}">
+  <h3>
+    {% blocktrans %}
+      The due date for this exam has passed
+    {% endblocktrans %}
+  </h3>
+  <p>
+    {% blocktrans %}
+      Since the due date for this exam has already passed, you are no longer be able to take this examination.
+    {% endblocktrans %}
+  </p>
+  <hr>
+  <p>
+    {% blocktrans %}
+      View your credit eligibility status on your <a href="{{progress_page_url}}">Progress</a> page.
+    {% endblocktrans %}
+  </p>
+</div>
+{% include 'proctored_exam/footer.html' %}
```

Comment:
@chrisndodge   I'd just change: "Since the due date for this exam has already passed, you are no longer able to take this examination."

to:

"Because the due date has passed, you are no longer able to take this exam."

(I try to avoid "since" as it can have two meanings)